docker/buoy/Dockerfile

```diff
@@ -0,0 +1,64 @@
+FROM ros:galactic-ros-base
+ENV DEBIAN_FRONTEND=noninteractive
+
+# Useful tools
+RUN apt update \
+ && apt install -y \
+        apt-utils \
+        build-essential \
+        curl \
+        lsb-release \
+        wget \
+ && apt clean
+
+# Setup and install ignition fortress
+RUN /bin/sh -c 'wget https://packages.osrfoundation.org/gazebo.gpg -O /usr/share/keyrings/pkgs-osrf-archive-keyring.gpg' \
+  && /bin/sh -c 'echo "deb [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/pkgs-osrf-archive-keyring.gpg] http://packages.osrfoundation.org/gazebo/ubuntu-stable $(lsb_release -cs) main" | tee /etc/apt/sources.list.d/gazebo-stable.list > /dev/null' \
+  && apt update \
+  && apt install -y ignition-fortress \
+  && rm -rf /var/lib/apt/lists/* \
+  && apt clean
+
+# Install colcon and rosdep
+RUN apt update \
+ && apt install -y \
```

Comment:
This command can be combined with the one above. It should save us some time if we don't clean and update all over again.